src/Symfony/Component/HttpFoundation/Tests/CookieTest.php

```diff
@@ -321,6 +346,9 @@ public function testFromString()
 
         $cookie = Cookie::fromString('foo_cookie=foo==; expires=Tue, 22 Sep 2020 06:27:09 GMT; path=/');
         $this->assertEquals(Cookie::create('foo_cookie', 'foo==', strtotime('Tue, 22 Sep 2020 06:27:09 GMT'), '/', null, false, false, true, null), $cookie);
```

Comment:
We should add `false` for the `$partitioned` argument in the current tests

Reply:
If we do so, I suggest an additional "minimal construction" test which compares a minimum object instantiation against a fully defined object (with the current default parameter values). This would cover modifications of the constructor's default parameter. Otherwise, all tests would still pass if you change a constructor's parameter default value since we overrule it in the tests. What do you think?

Reply:
Why not 👍

Reply:
added minimal constructor and `create` test to cover the "default cookie state".